Use of PyString_Check in flow python interpreter causes duplicate sym issues

```
./../../lib/libascent_flow.a(flow_python_interpreter.cpp.o): In function `PyString_Check(_object*)':
flow_python_interpreter.cpp:(.text+0x0): multiple definition of `PyString_Check(_object*)'
CMakeFiles/flow_python.dir/flow_python.cpp.o:flow_python.cpp:(.text+0x0): first defined here
../../../lib/libascent_flow.a(flow_python_interpreter.cpp.o): In function `PyString_AsString(_object*)':
flow_python_interpreter.cpp:(.text+0x20): multiple definition of `PyString_AsString(_object*)'
CMakeFiles/flow_python.dir/flow_python.cpp.o:flow_python.cpp:(.text+0x20): first defined here
../../../lib/libascent_flow.a(flow_python_interpreter.cpp.o): In function `PyString_FromString(char const*)':
flow_python_interpreter.cpp:(.text+0xa0): multiple definition of `PyString_FromString(char const*)'
CMakeFiles/flow_python.dir/flow_python.cpp.o:flow_python.cpp:(.text+0xa0): first defined here
../../../lib/libascent_flow.a(flow_python_interpreter.cpp.o): In function `PyString_AsString_Cleanup(char*)':
flow_python_interpreter.cpp:(.text+0xb0): multiple definition of `PyString_AsString_Cleanup(char*)'
CMakeFiles/flow_python.dir/flow_python.cpp.o:flow_python.cpp:(.text+0xb0): first defined here
../../../lib/libascent_flow.a(flow_python_interpreter.cpp.o): In function `PyInt_Check(_object*)':
flow_python_interpreter.cpp:(.text+0xc0): multiple definition of `PyInt_Check(_object*)'
CMakeFiles/flow_python.dir/flow_python.cpp.o:flow_python.cpp:(.text+0xc0): first defined here
../../../lib/libascent_flow.a(flow_python_interpreter.cpp.o): In function `PyInt_AsLong(_object*)':
flow_python_interpreter.cpp:(.text+0xe0): multiple definition of `PyInt_AsLong(_object*)'
CMakeFiles/flow_python.dir/flow_python.cpp.o:flow_python.cpp:(.text+0xe0): first defined here
```

--- a/src/libs/ascent/python/ascent_module/ascent_mpi_python.cpp
+++ b/src/libs/ascent/python/ascent_module/ascent_mpi_python.cpp
@@ -55,14 +55,14 @@ using namespace ascent;
 #if defined(IS_PY3K)
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyString_Check(PyObject *o)
 {
     return PyUnicode_Check(o);
 }
 
 //-----------------------------------------------------------------------------
-char *
+static char *
 PyString_AsString(PyObject *py_obj)
 {
     char *res = NULL;
@@ -94,14 +94,14 @@ PyString_AsString(PyObject *py_obj)
 }
 
 //-----------------------------------------------------------------------------
-PyObject *
+static PyObject *
 PyString_FromString(const char *s)
 {
     return PyUnicode_FromString(s);
 }
 
 //-----------------------------------------------------------------------------
-void
+static void
 PyString_AsString_Cleanup(char *bytes)
 {
     free(bytes);
@@ -109,14 +109,14 @@ PyString_AsString_Cleanup(char *bytes)
 
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyInt_Check(PyObject *o)
 {
     return PyLong_Check(o);
 }
 
 //-----------------------------------------------------------------------------
-long
+static long
 PyInt_AsLong(PyObject *o)
 {
     return PyLong_AsLong(o);

--- a/src/libs/ascent/python/ascent_module/ascent_python.cpp
+++ b/src/libs/ascent/python/ascent_module/ascent_python.cpp
@@ -56,14 +56,14 @@ using namespace ascent;
 #if defined(IS_PY3K)
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyString_Check(PyObject *o)
 {
     return PyUnicode_Check(o);
 }
 
 //-----------------------------------------------------------------------------
-char *
+static char *
 PyString_AsString(PyObject *py_obj)
 {
     char *res = NULL;
@@ -95,14 +95,14 @@ PyString_AsString(PyObject *py_obj)
 }
 
 //-----------------------------------------------------------------------------
-PyObject *
+static PyObject *
 PyString_FromString(const char *s)
 {
     return PyUnicode_FromString(s);
 }
 
 //-----------------------------------------------------------------------------
-void
+static void
 PyString_AsString_Cleanup(char *bytes)
 {
     free(bytes);
@@ -110,14 +110,14 @@ PyString_AsString_Cleanup(char *bytes)
 
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyInt_Check(PyObject *o)
 {
     return PyLong_Check(o);
 }
 
 //-----------------------------------------------------------------------------
-long
+static long
 PyInt_AsLong(PyObject *o)
 {
     return PyLong_AsLong(o);

--- a/src/libs/flow/flow_python_interpreter.cpp
+++ b/src/libs/flow/flow_python_interpreter.cpp
@@ -38,7 +38,7 @@ using namespace std;
 #if defined(IS_PY3K)
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyString_Check(PyObject *o)
 {
     return PyUnicode_Check(o);
@@ -46,7 +46,7 @@ PyString_Check(PyObject *o)
 
 
 //-----------------------------------------------------------------------------
-char *
+static char *
 PyString_AsString(PyObject *py_obj)
 {
     char *res = NULL;
@@ -78,42 +78,42 @@ PyString_AsString(PyObject *py_obj)
 }
 
 //-----------------------------------------------------------------------------
-PyObject *
+static PyObject *
 PyString_FromString(const char *s)
 {
     return PyUnicode_FromString(s);
 }
 
 //-----------------------------------------------------------------------------
-void
+static void
 PyString_AsString_Cleanup(char *bytes)
 {
     free(bytes);
 }
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyInt_Check(PyObject *o)
 {
     return PyLong_Check(o);
 }
 
 //-----------------------------------------------------------------------------
-long
+static long
 PyInt_AsLong(PyObject *o)
 {
     return PyLong_AsLong(o);
 }
 
 //-----------------------------------------------------------------------------
-long
+static long
 PyInt_AS_LONG(PyObject *o)
 {
     return PyLong_AS_LONG(o);
 }
 
 //-----------------------------------------------------------------------------
-PyObject *
+static PyObject *
 PyNumber_Int(PyObject *o)
 {
     return PyNumber_Long(o);

--- a/src/libs/flow/python/flow_python.cpp
+++ b/src/libs/flow/python/flow_python.cpp
@@ -55,14 +55,14 @@ using namespace flow;
 #if defined(IS_PY3K)
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyString_Check(PyObject *o)
 {
     return PyUnicode_Check(o);
 }
 
 //-----------------------------------------------------------------------------
-char *
+static char *
 PyString_AsString(PyObject *py_obj)
 {
     char *res = NULL;
@@ -94,14 +94,14 @@ PyString_AsString(PyObject *py_obj)
 }
 
 //-----------------------------------------------------------------------------
-PyObject *
+static PyObject *
 PyString_FromString(const char *s)
 {
     return PyUnicode_FromString(s);
 }
 
 //-----------------------------------------------------------------------------
-void
+static void
 PyString_AsString_Cleanup(char *bytes)
 {
     free(bytes);
@@ -109,14 +109,14 @@ PyString_AsString_Cleanup(char *bytes)
 
 
 //-----------------------------------------------------------------------------
-int
+static int
 PyInt_Check(PyObject *o)
 {
     return PyLong_Check(o);
 }
 
 //-----------------------------------------------------------------------------
-long
+static long
 PyInt_AsLong(PyObject *o)
 {
     return PyLong_AsLong(o);

